src/System.Management.Automation/engine/CommandCompletion/CompletionAnalysis.cs

```diff
@@ -970,6 +994,13 @@ internal List<CompletionResult> GetResultHelper(CompletionContext completionCont
                                         return result;
                                     }
                                     break;
+                                case TokenKind.LBracket:
```

Comment:
Can you please add a simple comment here to explain the scenario this `case` covers?